conf/application.conf

```diff
@@ -74,6 +74,7 @@ application {
     }
     ssoKey = "something secure"
   }
+  title = "webKnossos Local"
```

Comment:
```suggestion
  title = "webKnossos"
```

I think just webKnossos is fine as a default.

Reply:
Please also add the new config parameter to `WkConf.scala`, even if we don't use it from there. And mention it in the migrations, so that we don't forget to set it for the labs.